docs/modules/ROOT/pages/documentation/ecosystem/arquillian-containers/payara-micro-managed.adoc

```diff
@@ -90,17 +90,15 @@ To change this, provide java debug options to `cmdOptions`.
 |===
 
 These options can all be provided through either an environment variable or system property.
-The easiest way to provide these to the test instance is through the surefire plugin.
+The easiest way to provide these to the test instance is is by using an `arquillian.xml` file.
```

Comment:
To help people that are not so familiar with Arquillian so that they don't need to search Arquillian docs on how and where to place the arquillian.xml file:
```suggestion
The easiest way to provide these to the test instance is is by using an `arquillian.xml` file placed on the test classpath.
```